lib/dialects/postgres/data-types.js

```diff
@@ -31,40 +31,13 @@ module.exports = BaseTypes => {
    * @see oid here https://github.com/lib/pq/blob/master/oid/types.go
    */
 
-  BaseTypes.UUID.types.postgres = {
-    oids: [2950],
-    array_oids: [2951]
-  };
-
-  BaseTypes.CIDR.types.postgres = {
-    oids: [650],
-    array_oids: [651]
-  };
-
-  BaseTypes.INET.types.postgres = {
-    oids: [869],
-    array_oids: [1041]
-  };
-
-  BaseTypes.MACADDR.types.postgres = {
-    oids: [829],
-    array_oids: [1040]
-  };
-
-  BaseTypes.JSON.types.postgres = {
-    oids: [114],
-    array_oids: [199]
-  };
-
-  BaseTypes.JSONB.types.postgres = {
-    oids: [3802],
-    array_oids: [3807]
-  };
-
-  BaseTypes.TIME.types.postgres = {
-    oids: [1083],
-    array_oids: [1183]
-  };
+  BaseTypes.UUID.types.postgres = ['uuid'];
```

Comment:
This mapping doesn't serves any purpose, right? but leave it for now

Reply:
You are right, as of now it doesn't have any purpose. Originally I had two cases in mind. The first one was to allow overriding the default type parser(`this.lib.types.typeParser`) within sequelize. In this case you need the mapping to get the OID and set the parser. The second case was range autodetection, which requires base types to be mapped.